src/cpc/crtc.h

```diff
@@ -208,7 +208,7 @@ void	CRTC_SetRenderState(BOOL State);
 
 void	CRTC_SetTrueColourRender(BOOL State);
 
-enum
+typedef enum
 {
```

Comment:
seems you overlooked this one ?

Reply:
yup you're right and i did miss it! I've fixed that now, would you mind taking another look?

Thanks!